### PR TITLE
[Fix] error when branchName has '/' and bug of reload button

### PR DESCRIPTION
**Pull request by hellenGUAN**
# Description

- Fix the error of getting commits from selected branch when there is a '/' in the branch name
- Fix the bug of none response when pressing reload button
- Change the name of the button from `reload commits` to `Reload`

### CODE REVIEW

**Review by jn2657 — DISMISSED** [review]

**Review by vitachu0528 — DISMISSED** [review]

File: src/App/component/ComparisonPage.jsx [hunk 1]
```diff
@@ -116,18 +116,51 @@ function ComparisonPage(prop) {
...
+        .then(() => {
+          getCommitFromDBLeft(leftBranchSelected)
+          getCommitFromDBRight(rightBranchSelected)
+          setLoading(false)
+        })
+        .catch((e) => {
+          alert(e.response?.status)
+          console.error(e)
+        })
+    }
```
Comment (VincentETN) [inline src/App/component/ComparisonPage.jsx]:
```suggestion
    repo = githubRepo !== undefined ? githubRepo : gitlabRepo
    if (repo !== undefined) {
      const query = repo.url.split(repo.type + ".com/")[1]
      sendPVSBackendRequest('POST', `http://localhost:9100/pvs-api/${repo.type}/commits/${query}`)
        .then(() => {
          getCommitFromDBLeft(leftBranchSelected)
          getCommitFromDBRight(rightBranchSelected)
          setLoading(false)
        })
        .catch((e) => {
          alert(e.response?.status)
          console.error(e)
        })
    }
```

File: src/App/component/ComparisonPage.jsx [hunk 2]
```diff
@@ -116,18 +116,39 @@ function ComparisonPage(prop) {
...
+  }
+
   const getCommitFromDBLeft = (branch) => {
```
Comment (VincentETN) [inline src/App/component/ComparisonPage.jsx:141]: 裡面的可以跟前面一樣合併，然後既然是 get function，我會期待回傳資料而不是在裡面 set data，建議用 `setCommitListDataLeft(getCommitByBranch(leftBranch))` 的方式取資料，get commit 方式左右都一樣就不用寫兩個。

Comment (hellenGUAN) [reply src/App/component/ComparisonPage.jsx:141]: `setCommitListDataLeft(getCommitByBranch(leftBranch))` 的部分也許可以等 refactor 的時候所有檔案一起改，就不在這支 PR 單獨改

Comment (Xanonymous-GitHub) [reply src/App/component/ComparisonPage.jsx:141]: In https://github.com/SE-8-2021/frontend/pull/38#discussion_r779470634 , Suggest do not directly put a function call into another function call's parameter, split them may better.

**Review by VincentETN — APPROVED** [review]

**Review by Xanonymous-GitHub — APPROVED** [review]
LGTM

**Pull request merged**